examples/basic_scan_streamer_interface/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strings"
+
+	"github.com/Ullaakut/nmap/v2"
+)
+
+// CustomType is your custom type in code.
+// You just have to make it a Streamer.
+type CustomType struct {
+	nmap.Streamer
+	File string
+}
+
+// Write is a function that handles the normal nmap stdout
+func (c *CustomType) Write(d []byte) (int, error) {
+	var err error
```

Comment:
This variable is not used in this example, WDYT of just returning `nil` at the end?